packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -253,30 +230,48 @@ function ActionBarActions(props) {
 with the rest of the UI, directing a user’s focus to elsewhere in a view.
 
 ```tsx example
-<ActionBarContainer>
-  <ActionBar selectedItemCount={1}>
-    <Item key="edit">
-      Edit
-    </Item>
-    <Item key="copy">
-      Copy
-    </Item>
-    <Item key="delete">
-      Delete
-    </Item>
-  </ActionBar>
-</ActionBarContainer>
-<ActionBarContainer>
-<ActionBar isEmphasized selectedItemCount={1}>
-    <Item key="edit">
-      Edit
-    </Item>
-    <Item key="copy">
-      Copy
-    </Item>
-    <Item key="delete">
-      Delete
-    </Item>
-  </ActionBar>
-</ActionBarContainer>
+function Example({isEmphasized}) {
+  let items = [
+    {key: 0, name: 'Adobe Photoshop'},
+    {key: 1, name: 'Adobe Illustrator'},
+    {key: 2, name: 'Adobe XD'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([0]));
+
+  return (
+    <ActionBarContainer height={300} width="size-5000">
+      <ListView items={items} aria-label="ListView with action bar" selectionMode="multiple" selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys}>
+        {(item) => (
+          <Item>
+            {item.name}
+          </Item>
+        )}
+      </ListView>
+      <ActionBar
+        isEmphasized={isEmphasized}
+        selectedItemCount={selectedKeys === 'all' ? items.length : selectedKeys.size}
+        onAction={(key) => alert(`Performing ${key} action on ${selectedKeys.size} rows.`)}
+        onClearSelection={() => setSelectedKeys(new Set([]))}>
+        <Item key="edit">
+          <Edit />
+          <Text>Edit</Text>
+        </Item>
+        <Item key="copy">
+          <Copy />
+          <Text>Copy</Text>
+        </Item>
+        <Item key="delete">
+          <Delete />
+          <Text>Delete</Text>
+        </Item>
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<Flex direction="row" gap="size-200">
+  <Example />
+  <Example isEmphasized />
```

Comment:
Nit:
The example should have separate aria-labels to distinguish them from one another. One should mention it is a emphasized actionbar